### DISCUSSION TITLE
Features you would like to see?

### DISCUSSION THREAD

**Discussion by cyp3rius** [VirtusLab/strapi-plugin-navigation#145] [Ideas]
Hello Folks!

As we're already finished v4 re-implementation, it's time to ask you, the community, what features you're missing the most and would like to see in upcoming releases of the plugin? :)

We're going to build a community driven backlog combined with our own we're working on right now. Time start!

**Comment by edardev** [comment]
Hi @cyp3rius ,

First of all, thanks for your contributions to open source and the @strapi project. It's been a while since I last tried this plugin.
I'll let you know after I implement a new instance of @strapi. Some considerations for future versions if they are not supported yet, it will be good to integrate them with native @strapi plugins.

1. Multiple Menus
2. The Internationalization (i18n)
3. Graph API
4. Roles and Permissions

**Reply by cyp3rius** [reply]
Points 1, 3, 4 are already part of `2.0.0` :)

With i18n we are going to play for `2.1.0`

**Comment by konhi** [comment]
**Examples!**

Currently, I couldn't find any examples of front-end implementation with that plugin. It's really cool to show these things. I feel one of the selling-points of hyped frameworks like Next.js is that they have lots of starters and examples.

Additionaly, it might be cool to add a place for links to open-source projects which also are using the plugin!
I want to use this plugin in my project, so I can contribute to that.

**Reply by ltsNotMike** [reply]
Example with frontend integration is a work in progress. I'll be pleased to mention you in the PR, so you can give me some feedback on this or contribute. I will try to push what I have today tomorrow.

**Comment by galago** [comment]
We would like to see the possibility of rearranging the menu items to different levels, not just within its parent realm. 🙏
